src/coreclr/vm/eventtrace.cpp

```diff
@@ -3544,16 +3544,28 @@ VOID ETW::MethodLog::MethodJitted(MethodDesc *pMethodDesc, SString *namespaceOrC
                                         TRACE_LEVEL_INFORMATION,
                                         CLR_JITTEDMETHODILTONATIVEMAP_KEYWORD))
         {
+#ifdef FEATURE_CODE_VERSIONING
             ETW::MethodLog::SendMethodILToNativeMapEvent(pMethodDesc,
                                                          ETW::EnumerationLog::EnumerationStructs::JitMethodILToNativeMap,
                                                          pNativeCodeStartAddress,
                                                          pConfig->GetCodeVersion().GetVersionId(),
                                                          pConfig->GetCodeVersion().GetILCodeVersionId());
+#else
+            ETW::MethodLog::SendMethodILToNativeMapEvent(pMethodDesc,
+                                                         ETW::EnumerationLog::EnumerationStructs::JitMethodILToNativeMap,
+                                                         pNativeCodeStartAddress,
+                                                         pConfig->GetCodeVersion().GetVersionId(),
+                                                         0);
```

Comment:
Can we this ifdef smaller e.g. by storing the arg in a local and just ifdef the local?

Reply:
`GetILCodeVersionId` returns 0 when not `FEATURE_CODE_VERSIONING`, so these ifdefs were removed.